frontend/app/mocks/power-platform-api.server.ts

```diff
@@ -342,3 +373,59 @@ function getPersonalInformation(personalSinId: string) {
         where: { sinIdentification: { equals: personalSinId } },
       });
 }
+
+export function toPersonalInformationDB(personalInformation: PersonalInfo): ToPersonalInformationDBArgs {
+  return {
+    mailingAddressStreet: personalInformation.mailingAddress?.streetName,
+    mailingAddressSecondaryUnitText: personalInformation.mailingAddress?.secondAddressLine,
+    mailingAddressCityName: personalInformation.mailingAddress?.cityName,
+    mailingAddressProvince: personalInformation.mailingAddress?.provinceTerritoryStateId,
+    mailingAddressCountryReferenceId: personalInformation.mailingAddress?.countryId,
+    mailingAddressPostalCode: personalInformation.mailingAddress?.postalCode,
+    homeAddressStreet: personalInformation.homeAddress?.streetName,
+    homeAddressSecondaryUnitText: personalInformation.homeAddress?.secondAddressLine,
+    homeAddressCityName: personalInformation.homeAddress?.cityName,
+    homeAddressProvince: personalInformation.homeAddress?.provinceTerritoryStateId,
+    homeAddressCountryReferenceId: personalInformation.homeAddress?.countryId,
+    homeAddressPostalCode: personalInformation.homeAddress?.postalCode,
+    sameHomeAndMailingAddress: personalInformation.homeAndMailingAddressTheSame,
+    clientNumber: personalInformation.clientNumber,
+    clientId: personalInformation.clientId,
+    applicantId: personalInformation.applictantId,
+    applicantCategoryCode: personalInformation.applicantCategoryCode,
+    birthdate: personalInformation.birthDate,
+    lastName: personalInformation.lastName,
+    firstName: personalInformation.firstName,
+    emailAddressId: personalInformation.emailAddress,
+    primaryTelephoneNumber: personalInformation.primaryTelephoneNumber,
+    alternateTelephoneNumber: personalInformation.alternateTelephoneNumber,
+    preferredMethodCommunicationCode: personalInformation.preferredLanguageId,
+  };
+}
+
+interface ToPersonalInformationDBArgs {
```

Comment:
We can remove this `interface` and have Typescript infer the return type of the `toPersonalInformationDB(..)` function instead:

```typescript
export function toPersonalInformationDB(personalInformation: PersonalInfo) { .. }
```

Reply:
removed